ui/plugins/failure_summary/main.html

```diff
@@ -63,7 +63,20 @@
 
         </li>
 
-        <li ng-if="!tabs.failureSummary.is_loading && jobLogsAllParsed && bugSuggestionsLoaded && job_log_urls.length && suggestions.length == 0">
+        <span ng-if="errors.length > 0">
+          <span><b>Failed Buildbot Steps</b></span>
+          <li ng-repeat="error in errors">
+              <ul>
+                  <li>{{error.name}} : {{error.result}}, Check
+                    <a title="Open in Log Viewer"
+                       target="_blank"
+                       href="{{error.lvURL}}">here</a>
```

Comment:
This is really awesome!  But I might change the text to just be "View log" to make it a bit more explicit and still terse.